src/pykeen/evaluation/evaluator.py

```diff
@@ -381,16 +377,13 @@ def _param_size_search(
         return cast(Tuple[int, bool], (values_dict[key], evaluated_once))
 
     @staticmethod
-    def _check_slicing_availability(model: Model, batch_size: int) -> None:
+    def _check_slicing_availability(model: Model, batch_size: int, entities: bool, relations: bool) -> None:
```

Comment:
can you add a default value for entities and relations as well as add a docstring with explanations of all parameters?

Reply:
this is a private static method only used once; I'll add doc, but I am not sure whether we should support not providing all parameters explicitly, as this might shadow (internal) errors

Reply:
doc in d72895e